lib/matplotlib/table.py

```diff
@@ -558,6 +590,10 @@ def table(ax,
           rowLabels=None, rowColours=None, rowLoc='left',
           colLabels=None, colColours=None, colLoc='center',
           loc='bottom', bbox=None, edges='closed',
+          cellEdgeColour=None,
```

Comment:
I really don't see how colors.to_rgba_array helps here.

I've simply removed the singular version.

Note that current code doesn't implicitly broadcase rowColours or colColours -- you get an exception if a single value is passed.